### ISSUE TITLE
Inline guidance: research plan

### ISSUE THREAD

**Issue by rachel-kauff** [department-of-veterans-affairs/va.gov-cms#8749]
We've developed hypothesis around providing inline guidance to editors. Before we can move forward with implementation and further content development, we should validate them with users. Ideally, we should test them with the users who will benefit the most from having better cms guidance. We believe these are editors in the field, either VAMC or Vet Center.

ACs
- [x] Research plan is written following [the VA research template](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/research/research-plan-template.md)
- [x] Plan is validated with PO
- [x] Plan includes involving VHA digital media in supporting recruitment efforts
- [x] Plan includes validating guidance content with the Sitewide Content team

**Comment by rachel-kauff** [comment]
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/cms/research/inline-guidance/inline-guidance-research-plan.md

**Comment by rachel-kauff** [comment]
The PO validation piece of this ticket will need to get pushed to early next sprint due to Dave's PTO. I've scheduled a review meeting w/ Dave and Erika for 5/10.

**Comment by rachel-kauff** [comment]
Please note: We're refraining from investing time in finalizing a prototype and moderation guide until we get research approval.

**Comment by EWashb** [comment]
PO has given direction. We must sync with research ops and then determine the best group for this effort. PO suggests looking at the R&S pilot for users. Also would be advantageous to consider Vet Center editors since we have many new editors coming on board.

**Comment by EWashb** [comment]
Pushed to the next sprint to give me time to coordinate with VA Research Ops group.

**Comment by rachel-kauff** [comment]
Needs updating per the pivot. Will tackle on 6/17

**Comment by EWashb** [comment]
@rachel-kauff should we refine this one again with PO? I can add it to sprint 61 and maybe we will have some time to get his insight in our planning meeting

**Comment by rachel-kauff** [comment]
Yes, I reckon so. I set up a meeting for that. I can give it a first pass on Tues.

**Comment by EWashb** [comment]
@rachel-kauff I feel like this is ready to close, yes?

**Comment by EWashb** [comment]
This is currently on hold to account for resourcing needs.

**Comment by valerierunde** [comment]
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/cms/research/inline-guidance/inline-guidance-research-plan.md
